tests/unit/tickstore/test_toplevel.py

```diff
@@ -131,6 +132,8 @@ def test_slice_list_of_dicts(start, end, expected_start_index, expected_end_inde
     assert expected == result
 
 
+@pytest.mark.xfail(sys.version_info >= (3,),
+                   reason="python3 issue with mock 1.0.1")
```

Comment:
There are a bunch of these mock issues - can we resolve before merge so we don't lose these tests?